client/lib/invites/stores/invite-accepted.js

```diff
@@ -8,13 +8,14 @@ const InviteMessageStore = createReducerStore( ( state, payload ) => {
 	const { action } = payload;
 	let newState = Object.assign( {}, state );
 	switch ( action.type ) {
+		case ActionTypes.INVITE_ACCEPTED:
 		case ActionTypes.DISPLAY_INVITE_ACCEPTED_NOTICE:
 			newState.accepted = true;
-			newState.siteId = action.siteId;
+			newState.siteId = parseInt( action.invite.site.ID );
 			return newState;
 		case ActionTypes.DISPLAY_INVITE_DECLINED_NOTICE:
 			newState.declined = true;
-			newState.siteId = action.siteId;
+			newState.siteId = parseInt( action.siteId );
```

Comment:
Note that on this line and line `14`, we should be passing `10` as the second parameter to `parseInt()`.